include/Decimal.hpp

```diff
@@ -198,6 +202,15 @@ class Decimal {
         min.SetMin();
         return getDecimal() == min;
     }
+
+    bool operator==(const Decimal &other) {
+        return ::memcmp((void *) m_data, other.m_data, sizeof(m_data)) == 0;
```

Comment:
Tested it for signed zero equality, it worked. Have updated == operator for decimal class to use ttmath::Int == operator (ttmath::Int is the data storage for decimal data). Thanks Chris for pointing out about signed zero scenario.